www/front_src/src/Resources/columns/State/DetailsTable/index.tsx

```diff
@@ -69,24 +69,30 @@ const DetailsTable = <TDetails extends unknown>({
           </TableRow>
         </TableHead>
         <TableBody>
-          <TableRow>
-            {loading && (
+          {loading && (
+            <TableRow>
               <TableCell colSpan={columns.length}>
                 <Skeleton height={20} animation="wave" />
               </TableCell>
-            )}
-            {success &&
-              columns.map(({ label, getFormattedString }) => (
-                <TableCell key={label}>
-                  <span>{getFormattedString?.(details)}</span>
-                </TableCell>
-              ))}
-            {error && (
+            </TableRow>
+          )}
+          {success &&
+            details?.map((detail) => (
```

Comment:
details could be initialize to [] to avoid optional chaining

Reply:
Right now it wouldn't work, because 'null' is the error case and is a potential value; this makes TypeScript prevent us from calling map on details without error.

I'll refactor the DetailsTable in another PR to clean things up.